Dockerfile

```diff
@@ -1,28 +1,36 @@
 # Based on https://github.com/sclorg/s2i-nodejs-container
 FROM quay.io/sclorg/nodejs-18-c8s
-ARG ADDPKGS=""
-ARG NPMLOCATION="open"
+
+# Additional packages to install before the build.
+ARG ADDPKGS=
+# npm mirror to use to install dependencies.
+ARG NPMLOCATION=open
+
 USER root
-RUN yum install -y "krb5-workstation" "postgresql" $ADDPKGS && \
-    yum clean all -y
-COPY rhcachain.crt "$HOME/"
-RUN trust anchor --store "${HOME}/rhcachain.crt"
-# OSCI-2964
-RUN sed -i -e '/dns_canonicalize_hostname/d;/^\[libdefaults\]/a\ \ dns_canonicalize_hostname = fallback' /etc/krb5.conf
-COPY "src" "$HOME/src/"
-COPY "assets" "$HOME/assets/"
-COPY "package.json" "package-lock.json" "env.sh" "tsconfig.json" "$HOME/"
-RUN echo "Use location: $NPMLOCATION"
-COPY ".npmrcs/$NPMLOCATION" ".npmrc"
+RUN dnf install --assumeyes krb5-workstation postgresql $ADDPKGS && \
+    dnf clean all --assumeyes && \
+    # OSCI-2964 \
+    sed --in-place -e '/dns_canonicalize_hostname/d;/^\[libdefaults\]/a\ \ dns_canonicalize_hostname = fallback' /etc/krb5.conf
+
+COPY rhcachain.crt ./
+RUN trust anchor --store "$HOME/rhcachain.crt"
+
+COPY assets/ src/ package.json package-lock.json env.sh tsconfig.json $HOME/
```

Comment:
This will not work.
It will copy content of the dirs to the target.
In the current Dockerfile directories are kept.

Reply:
Good catch. Fixed now.